### ISSUE TITLE
Cant open with KiCad

### ISSUE THREAD

**Issue by Missingo1996** [librespacefoundation/upsat-adcs-hardware#6]
I tried it with KiCad too, it doenst work either.
Is it possible to provide a PDF Version of the schematic?

**Comment by ppapadeas** [comment]
Hey, sorry for the confusion. This repo is made with gEDA.

**Comment by Missingo1996** [comment]
Can you distribute an PDF Version please?
Cause I dont have a Linux System respectively gEDA.

**Comment by ppapadeas** [comment]
![spint](https://cloud.githubusercontent.com/assets/498592/21421801/6e93ae70-c83d-11e6-959f-dc6d789dbdf0.png)
![sensors](https://cloud.githubusercontent.com/assets/498592/21421803/6e97183a-c83d-11e6-924a-abddb416c19a.png)
![photodiodes](https://cloud.githubusercontent.com/assets/498592/21421805/6ea1ed78-c83d-11e6-8853-5c4a2f1a9877.png)
![main](https://cloud.githubusercontent.com/assets/498592/21421804/6e99f528-c83d-11e6-936d-0183ba695bfc.png)
![magnetodriver](https://cloud.githubusercontent.com/assets/498592/21421802/6e95767e-c83d-11e6-91c4-81b3e241dc64.png)
![adcs](https://cloud.githubusercontent.com/assets/498592/21421800/6e92a052-c83d-11e6-9acc-9e966c4f196d.png)

**Comment by ppapadeas** [comment]
On driving the coil @zisi or @ykoveos can answer that! :)

**Comment by zisi** [comment]
Hey, we measure the resistance of coil, we know the voltage (PWM) that applied in coil so by using ohm's law we calculate the current. Take a look at code [here](https://github.com/librespacefoundation/upsat-adcs-software/blob/master/adcs/Src/adcs_actuators.c)

**Comment by ykoveos** [comment]
Hi, 
yes coil resistance and 5V rail is pretty stable for our purposes, for testing we can query 5V current from EPS before and after energizing a coil. The controller is robust enough to its gain.

**Comment by Missingo1996** [comment]
@zisi @ykoveos Okay, that makes sense. But isnt there a resistance fluctuation because of the temperature fluctuation?
I do make an magnettorquer as a school project and my idea was to use an I2C Motor Driver and Shunt to measure the current but I dont know if this is an good idea.

**Comment by ykoveos** [comment]
Yes for copper coils  is about 20% for 50C (with no compensation) but the controller is tested for changes in gain  for about two orders of magnitude (10000%). Essentially the controller gain embeds the voltage to current coefficient (R).
If you need to measure current for each coil (which is indeed a necessary feature for what you want to evaluate) i would suggest using  MAX44284 for analog output or INA226 for I2C. 
If you need more info on this send PM
